docs/add-ins/get-or-set-the-location-of-an-appointment.md

```diff
@@ -1,30 +1,111 @@
+- [item.location](/office/dev/add-ins/reference/objectmodel/requirement-set-1.1/office.context.mailbox.item#location-stringlocation): Basic API that allows you to get and set the location.
+- [item.enhancedLocation](/office/dev/add-ins/reference/objectmodel/requirement-set-1.8/office.context.mailbox.item#enhancedlocation-enhancedlocation): Enhanced API that allows you to get and set the location, and includes specifying the [location type](/javascript/api/outlook/office.mailboxenums.locationtype?view=outlook-js-1.8). The type is `LocationType.Custom` if you set the location using `item.location`.
 
 The following table lists the location APIs and the modes (i.e., Compose or Read) where they are available.
 
 | API | Applicable appointment modes |
 |---|---|
-| [item.location](/javascript/api/outlook/office.appointmentread#location) | Attendee/Read |
-| [item.location.getAsync](/javascript/api/outlook/office.location#getasync-options--callback-) | Organizer/Compose |
-| [item.location.setAsync](/javascript/api/outlook/office.location#setasync-location--options--callback-) | Organizer/Compose |
-| [item.enhancedLocation.getAsync](/javascript/api/outlook/office.enhancedlocation#getasync-options--callback-) | Organizer/Compose,<br>Attendee/Read |
-| [item.enhancedLocation.addAsync](/javascript/api/outlook/office.enhancedlocation#addasync-locationidentifiers--options--callback-) | Organizer/Compose |
-| [item.enhancedLocation.removeAsync](/javascript/api/outlook/office.enhancedlocation#removeasync-locationidentifiers--options--callback-) | Organizer/Compose |
+| [item.location](/javascript/api/outlook/office.appointmentread?view=outlook-js-1.1#location) | Attendee/Read |
+| [item.location.getAsync](/javascript/api/outlook/office.location?view=outlook-js-1.1#getasync-options--callback-) | Organizer/Compose |
+| [item.location.setAsync](/javascript/api/outlook/office.location?view=outlook-js-1.1#setasync-location--options--callback-) | Organizer/Compose |
+| [item.enhancedLocation.getAsync](/javascript/api/outlook/office.enhancedlocation?view=outlook-js-1.8#getasync-options--callback-) | Organizer/Compose,<br>Attendee/Read |
+| [item.enhancedLocation.addAsync](/javascript/api/outlook/office.enhancedlocation?view=outlook-js-1.8#addasync-locationidentifiers--options--callback-) | Organizer/Compose |
+| [item.enhancedLocation.removeAsync](/javascript/api/outlook/office.enhancedlocation?view=outlook-js-1.8#removeasync-locationidentifiers--options--callback-) | Organizer/Compose |
```

Comment:
Remove the view parameters from all these. We shouldn't be sending users to 1.1.